src/ui/states/state-success.svelte

```diff
@@ -14,5 +18,7 @@
   {onContinue}
 >
   <IconSuccess slot="icon" />
-  Your plan is now active.
+  {#if isSubscription}
+    Your plan is now active.
```

Comment:
I'm wondering if this should also be shown for non-consumables... If they are used as lifetimes, I think it could make kind of sense, but might be safer to not show it to cover more scenarios... cc @jefago